### ISSUE TITLE
fake.build domain expires 2022-10-28

### ISSUE THREAD

**Issue by matthid** [fsprojects/FAKE#2682]
This is an announcement to let you guys know that I cancelled the domain renewal. As I'm not involved in the project anymore it didn't make a lot of sense for me to own that domain. 
Imho we have two ways forward, 
- either someone or some business steps in to fund the domain for the project 
- or we just use GitHub pages in the future, for this we need to remove the current redirect. Once the redirect is removed I can enable a redirect back to GitHub pages until it expires.

I hope the transition is not too painful...

**Comment by falfaddaghi** [comment]
I am willing to fund the domain.
if we decide to move to github pages I would rather not have it not have a deadline.
@matthid  let me know how you want to transfer the domain.

**Comment by matthid** [comment]
@falfaddaghi Thanks a lot! 
Yes, tbh I wasn't sure about the best way forward, that's why I opened the issue. 
Do you need anything else than the domain authorization code? I guess I'll file a secure way to send it to you?

**Comment by falfaddaghi** [comment]
can we setup a quick meeting to handle this?
can i send the invite to the same email? you used before?

**Comment by matthid** [comment]
Yeah sure just send me an invite, I'll answer per mail if the time works out.
